dockstore-webservice/src/main/java/io/dockstore/webservice/resources/DockerRepoResource.java

```diff
@@ -175,40 +182,34 @@ public Container getContainer(@ApiParam(hidden = true) @Auth Token authToken,
     @Timed
     @UnitOfWork
     @Path("/{containerId}/labels")
-    @ApiOperation(value = "Update the labels linked to a container.",
-		notes = "Labels are alphanumerical (case-insensitive and may contain internal hyphens), given in a comma-delimited list.",
-		response = Container.class)
+    @ApiOperation(value = "Update the labels linked to a container.", notes = "Labels are alphanumerical (case-insensitive and may contain internal hyphens), given in a comma-delimited list.", response = Container.class)
     public Container updateLabels(@ApiParam(hidden = true) @Auth Token authToken,
-            @ApiParam(value = "Container to modify.", required = true)
-    			@PathParam("containerId") Long containerId,
-            @ApiParam(value = "Comma-delimited list of labels.", required = true)
-    			@QueryParam("labels") String labelStrings) {
+            @ApiParam(value = "Container to modify.", required = true) @PathParam("containerId") Long containerId,
+            @ApiParam(value = "Comma-delimited list of labels.", required = true) @QueryParam("labels") String labelStrings) {
         Container c = containerDAO.findById(containerId);
         Helper.checkContainer(c);
-        
+
         if (labelStrings.length() == 0) {
-        	c.setLabels(new HashSet<>(0));
+            c.setLabels(new HashSet<>(0));
         } else {
-        	Set<String> labelStringSet = new HashSet<String>(
-        			Arrays.asList(labelStrings.toLowerCase().split("\\s*,\\s*")));
+            Set<String> labelStringSet = new HashSet<String>(Arrays.asList(labelStrings.toLowerCase().split("\\s*,\\s*")));
             final String labelStringPattern = "^[a-zA-Z0-9]+(-[a-zA-Z0-9]+)*$";
             final int labelMaxLength = 255;
```

Comment:
add a comment that this matches the restriction on labels to 255 characters probably and that if this is changed then the java object/mapped db schema needs to be changed